codec-http/src/main/java/io/netty/handler/codec/http/HttpObjectAggregator.java

```diff
@@ -48,8 +49,16 @@
  * ...
  * p.addLast("handler", new HttpRequestHandler());
  * </pre>
- * Be aware that you need to have the {@link HttpResponseEncoder} or {@link HttpRequestEncoder}
- * before the {@link HttpObjectAggregator} in the {@link ChannelPipeline}.
+ * Be aware that you need to have the {@link HttpResponseDecoder} or {@link HttpRequestDecoder}
+ * before the {@link HttpObjectAggregator} in the {@link ChannelPipeline} for it to receive
+ * {@link HttpObject}s to aggregate, and you need a {@link HttpResponseEncoder} before the
+ * {@link HttpObjectAggregator} if it will be receiving {@link HttpRequest} messages as the
+ * {@link HttpObjectAggregator} may write a 100 Continue response.
```

Comment:
> Isn't that true though?

`HttpObjectAggregator` sending these responses is a side effect of its primary responsibility to aggregate all `HttpObjects` into a `FullHttpMessage`.  So I don't want the comment to imply that in order to have these responses sent you need `HttpObjectAggregator` ... you can not aggregate (or aggregate in a different way) and still send these response types. The main take away is while `HttpObjectAggregator` is in the pipeline it has the responsibility to send these responses.

> and instead focus on what else HttpObjectAggregator can do for you

Yes this sounds good.